usaspending_api/common/tests/integration/test_decorators.py

```diff
@@ -66,16 +61,17 @@ def test_statement_timeout_no_decorator():
     """Test the django statement timeout setting"""
 
     start = perf_counter()
-    pg_sleep_in_seconds = 5
+    pg_sleep_in_seconds = 2
+    tiny_offset = 0.001  # allows for slight rounding or timing differences between tools
 
     def test_timeout_success():
         with connection.cursor() as cursor:
             # pg_sleep takes in a parameter corresponding to seconds
-            cursor.execute("SELECT pg_sleep({:d})".format(pg_sleep_in_seconds))
+            cursor.execute(f"SELECT pg_sleep({pg_sleep_in_seconds:.2f})")
 
     try:
         test_timeout_success()
     except Exception:
         assert False
     else:
-        assert (perf_counter() - start) >= pg_sleep_in_seconds
+        assert (perf_counter() - start) >= (pg_sleep_in_seconds - tiny_offset)
```

Comment:
Fixed this bothersome test from being _ever so slightly_ less than 5s

Reply:
Does this have anything to do with https://federal-spending-transparency.atlassian.net/browse/DEV-5953?

Reply:
More of a local problem. It might have failed once or twice in Travis over the past several years but it would constantly fail for me locally. So I would say "no"